pages/identities.tsx

```diff
@@ -8,12 +8,13 @@ import {
 } from "@starknet-react/core";
 import { useEffect, useState } from "react";
 import IdentitiesGallery from "../components/identities/identitiesGalleryV1";
-import MintIdentity from "../components/identities/mintIdentity";
+import MintIcon from "../components/UI/iconsComponents/icons/mintIcon";
 import { useRouter } from "next/router";
 import { hexToDecimal } from "../utils/feltService";
 import IdentitiesSkeleton from "../components/identities/identitiesSkeleton";
 import TxConfirmationModal from "../components/UI/txConfirmationModal";
 import Wallets from "../components/UI/wallets";
+import ClickableAction from "../components/UI/iconsComponents/clickableAction";
 
 const Identities: NextPage = () => {
   const { account, address } = useAccount();
```

Comment:
You can add the fix made on the identity page here pls ? https://github.com/starknet-id/app.starknet.id/pull/424/files#diff-2f396cbcb2a07a00a226255f1fc151a03e18ef663d722f673002730415d82f4dR20

You'll be able to close the issue #423